aviatrix/resource_aviatrix_gateway.go

```diff
@@ -1013,6 +1018,7 @@ func resourceAviatrixGatewayRead(d *schema.ResourceData, meta interface{}) error
 					d.Set("peering_ha_subnet", gwHaGw.VpcNet)
 				} else if gwHaGw.CloudType == goaviatrix.GCP {
 					d.Set("peering_ha_zone", gwHaGw.GatewayZone)
+					d.Set("peering_ha_subnet", gwHaGw.GatewaySubnet)
```

Comment:
Wouldn't this cause a diff if peering_ha_subnet is unset?

Reply:
Yes, actually I think there are 2 bugs here. 
1. This would cause a diff like you said.
2. I am using the wrong field from the gw struct

I will push a fix